app.py

```diff
@@ -28,34 +32,31 @@
 NACHET_DATA = os.getenv("NACHET_DATA")
 NACHET_MODEL = os.getenv("NACHET_MODEL")
 
+try:
+    VALID_EXTENSION = json.loads(os.getenv("NACHET_VALID_EXTENSION"))
+    VALID_DIMENSION = json.loads(os.getenv("NACHET_VALID_DIMENSION"))
+except TypeError:
+    # For testing
+    VALID_DIMENSION = {"width": 1920, "height": 1080}
+    VALID_EXTENSION = {"jpeg", "jpg", "png", "gif", "bmp", "tiff", "webp"}
+
 CACHE = {
     'seeds': None,
-    'endpoints': None
+    'endpoints': None,
+    'validators': []
 }
 
-# Check: do environment variables exist?
-if connection_string is None:
-    raise ServerError("Missing environment variable: NACHET_AZURE_STORAGE_CONNECTION_STRING")
-
-if endpoint_url is None:
-    raise ServerError("Missing environment variable: NACHET_MODEL_ENDPOINT_REST_URL")
-
-if endpoint_api_key is None:
-    raise ServerError("Missing environment variables: NACHET_MODEL_ENDPOINT_ACCESS_KEY")
-
-# Check: are environment variables correct?
-if not bool(re.match(connection_string_regex, connection_string)):
-    raise ServerError("Incorrect environment variable: NACHET_AZURE_STORAGE_CONNECTION_STRING")
-
-if not bool(re.match(endpoint_url_regex, endpoint_url)):
-    raise ServerError("Incorrect environment variable: NACHET_MODEL_ENDPOINT_ACCESS_KEY")
-
 app = Quart(__name__)
 app = cors(app, allow_origin="*", allow_methods=["GET", "POST", "OPTIONS"])
-mb = int(os.getenv("NACHET_MAX_CONTENT_LENGTH"))
+
+try:
+    mb = int(os.getenv("NACHET_MAX_CONTENT_LENGTH"))
```

Comment:
ok, we probably want to have this named NACHET_MAX_CONTENT_LENGTH_MEGABYTES as I'd need to read the source code to figure out which unit it is. typically we expect K for sizes, but for images for sure megabytes make more sense but it won't be obvious what unit it is.

Reply:
Done with MAX_CONTENT_LENGTH